realm/realm-library/src/main/java/io/realm/rx/RealmObservableFactory.java

```diff
@@ -29,331 +35,337 @@
 import io.realm.RealmObject;
 import io.realm.RealmQuery;
 import io.realm.RealmResults;
-import rx.Observable;
-import rx.Subscriber;
-import rx.functions.Action0;
-import rx.subscriptions.Subscriptions;
-
 
 /**
- * Factory class for creating Observables for RxJava (&lt;=1.1.*).
+ * Factory class for creating Observables for RxJava (&lt;=2.0.*).
  *
- * @see Realm#asObservable()
- * @see RealmObject#asObservable()
- * @see RealmResults#asObservable()
- * @see DynamicRealm#asObservable()
- * @see DynamicRealmObject#asObservable()
+ * @see Realm#asFlowable() ()
+ * @see RealmObject#asFlowable()
+ * @see RealmResults#asFlowable()
+ * @see DynamicRealm#asFlowable()
+ * @see DynamicRealmObject#asFlowable()
  */
 public class RealmObservableFactory implements RxObservableFactory {
 
     // Maps for storing strong references to Realm classes while they are subscribed to.
     // This is needed if users create Observables without manually maintaining a reference to them.
     // In that case RealmObjects/RealmResults/RealmLists might be GC'ed too early.
-    ThreadLocal<StrongReferenceCounter<RealmResults>> resultsRefs = new ThreadLocal<StrongReferenceCounter<RealmResults>>() {
+    private ThreadLocal<StrongReferenceCounter<RealmResults>> resultsRefs = new ThreadLocal<StrongReferenceCounter<RealmResults>>() {
         @Override
         protected StrongReferenceCounter<RealmResults> initialValue() {
             return new StrongReferenceCounter<RealmResults>();
         }
     };
-    ThreadLocal<StrongReferenceCounter<RealmList>> listRefs = new ThreadLocal<StrongReferenceCounter<RealmList>>() {
+    private ThreadLocal<StrongReferenceCounter<RealmList>> listRefs = new ThreadLocal<StrongReferenceCounter<RealmList>>() {
         @Override
         protected StrongReferenceCounter<RealmList> initialValue() {
             return new StrongReferenceCounter<RealmList>();
         }
     };
-    ThreadLocal<StrongReferenceCounter<RealmModel>> objectRefs = new ThreadLocal<StrongReferenceCounter<RealmModel>>() {
+    private ThreadLocal<StrongReferenceCounter<RealmModel>> objectRefs = new ThreadLocal<StrongReferenceCounter<RealmModel>>() {
         @Override
         protected StrongReferenceCounter<RealmModel> initialValue() {
             return new StrongReferenceCounter<RealmModel>();
         }
     };
 
+    private static final BackpressureStrategy BACK_PRESSURE_STRATEGY = BackpressureStrategy.LATEST;
```

Comment:
@curiousily it is the only backpressure strategy that makes sense for RealmResults. It will always show its latest value, why enqueue events of the same thing?